olm-catalog/serverless-operator/1.5.0/serving_v1alpha1_knativeserving_crd.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: knativeservings.serving.knative.dev
```

Comment:
i thought the plan was to remove this starting with 1.5.0

Reply:
There is no accepted plan AFAIK and I didn't concern myself with this decision for this PR. If we're removing it, I'd rather consciously do so in a separate PR that states that we're removing it. I'm happy if we are but I don't want us to block testing on 0.12 by that decision as @nak3 needs it for the Kourier migration work. It's orthogonal.